tests/framework/CodeInterfaceTests/AbceInterfaceTest/inputs/ts_data/timeseries_load_hourly.csv

```diff
@@ -0,0 +1,8785 @@
+10/18/2020 14:00,0.497005071,15897.9
+10/18/2020 15:00,0.495817103,15859.9
+10/18/2020 16:00,0.503713962,16112.5
+10/18/2020 17:00,0.51791643,16566.8
+10/18/2020 18:00,0.52241195,16710.6
+10/18/2020 19:00,0.516137604,16509.9
+10/18/2020 20:00,0.4963173,15875.9
+10/18/2020 21:00,0.46572713,14897.4
+10/18/2020 22:00,0.434608627,13902
+10/18/2020 23:00,0.408857863,13078.3
+10/19/2020 0:00,0.392129401,12543.2
+10/19/2020 1:00,0.384357591,12294.6
+10/19/2020 2:00,0.381203224,12193.7
+10/19/2020 3:00,0.38919387,12449.3
+10/19/2020 4:00,0.408639027,13071.3
+10/19/2020 5:00,0.450308559,14404.2
+10/19/2020 6:00,0.487638883,15598.3
+10/19/2020 7:00,0.500218836,16000.7
+10/19/2020 8:00,0.502560383,16075.6
+10/19/2020 9:00,0.507115302,16221.3
+10/19/2020 10:00,0.506586969,16204.4
+10/19/2020 11:00,0.503182503,16095.5
+10/19/2020 12:00,0.499096519,15964.8
+10/19/2020 13:00,0.494132065,15806
+10/19/2020 14:00,0.488639277,15630.3
+10/19/2020 15:00,0.484518904,15498.5
+10/19/2020 16:00,0.490011692,15674.2
+10/19/2020 17:00,0.505836673,16180.4
+10/19/2020 18:00,0.508834729,16276.3
+10/19/2020 19:00,0.496536136,15882.9
+10/19/2020 20:00,0.480992516,15385.7
+10/19/2020 21:00,0.460390654,14726.7
+10/19/2020 22:00,0.429928659,13752.3
+10/19/2020 23:00,0.406113032,12990.5
+10/20/2020 0:00,0.388631149,12431.3
+10/20/2020 1:00,0.379737021,12146.8
+10/20/2020 2:00,0.375147714,12000
+10/20/2020 3:00,0.373400151,11944.1
+10/20/2020 4:00,0.38448264,12298.6
+10/20/2020 5:00,0.405018851,12955.5
+10/20/2020 6:00,0.419940352,13432.8
+10/20/2020 7:00,0.431332337,13797.2
+10/20/2020 8:00,0.438791524,14035.8
+10/20/2020 9:00,0.443568405,14188.6
+10/20/2020 10:00,0.441070547,14108.7
+10/20/2020 11:00,0.435358923,13926
+10/20/2020 12:00,0.428115445,13694.3
+10/20/2020 13:00,0.421719177,13489.7
+10/20/2020 14:00,0.415225995,13282
+10/20/2020 15:00,0.417161132,13343.9
+10/20/2020 16:00,0.420093537,13437.7
+10/20/2020 17:00,0.442224126,14145.6
+10/20/2020 18:00,0.45508544,14557
+10/20/2020 19:00,0.449405078,14375.3
+10/20/2020 20:00,0.44166453,14127.7
+10/20/2020 21:00,0.423029068,13531.6
+10/20/2020 22:00,0.403615173,12910.6
+10/20/2020 23:00,0.384013705,12283.6
+10/21/2020 0:00,0.371558801,11885.2
+10/21/2020 1:00,0.359041373,11484.8
+10/21/2020 2:00,0.358481777,11466.9
+10/21/2020 3:00,0.357919056,11448.9
+10/21/2020 4:00,0.363380581,11623.6
+10/21/2020 5:00,0.372371621,11911.2
+10/21/2020 6:00,0.380577978,12173.7
+10/21/2020 7:00,0.387039897,12380.4
+10/21/2020 8:00,0.397059467,12700.9
+10/21/2020 9:00,0.402989927,12890.6
+10/21/2020 10:00,0.404177895,12928.6
+10/21/2020 11:00,0.406672627,13008.4
+10/21/2020 12:00,0.406328742,12997.4
+10/21/2020 13:00,0.405862934,12982.5
+10/21/2020 14:00,0.405081376,12957.5
+10/21/2020 15:00,0.407891857,13047.4
+10/21/2020 16:00,0.417036083,13339.9
+10/21/2020 17:00,0.444221787,14209.5
+10/21/2020 18:00,0.460922113,14743.7
+10/21/2020 19:00,0.457458249,14632.9
+10/21/2020 20:00,0.445813039,14260.4
+10/21/2020 21:00,0.419752778,13426.8

```

Comment:
Hm, if it is easy, reducing the size of these files would be better, but if not, this is fine. (Such as by only having a months worth of data)